tsconfig.json

```diff
@@ -38,8 +38,9 @@
     "*.{ts|tsx|js|jsx}",
     "scripts/*",
     "test/*",
-    "cypress/*",
-    ".storybook/*"
+    "cypress/**/*",
+    ".storybook/*",
+    "*.ts"
```

Comment:
Do we need this `"*.ts"`? Isn't it captured in line 38? (`"*.{ts|tsx|js|jsx}"`)

Reply:
no we don't & you're right it is, leftover weirdness. will remove